docs/source/api/shape_utils.rst

```diff
@@ -16,3 +16,4 @@ This module introduces functions that are made aware of the requested `size_tupl
    to_tuple
    rv_size_is_none
    change_dist_size
+   broadcast_dist_samples_shape
```

Comment:
We don't need this is the user-facing docs, it's a legacy method that we want to get rid of